.travis.yml

```diff
@@ -1,10 +1,14 @@
 sudo: required
 language: node_js
 matrix:
+  fast_finish: true
```

Comment:
i dont think this works the way you think it does.

i only know this because everytime i see this property i get excited, but then read the docs and get disappointed: https://docs.travis-ci.com/user/customizing-the-build/#Fast-Finishing

it needs `allow_failures` marked on some jobs

Reply:
actually I do :p but I saw so many projects using fast_finish without allow_failures that I thought travis might haven't documented everything :p but i'll revert the commit.